Update 05_progettazione-dei-servizi-digitali.rst (#224)
italia/lg-modellointeroperabilita-docs: doc/00_Linee guida interoperabilità tecnica/05_progettazione-dei-servizi-digitali.rst

> Progettazione dei servizi digitali e degli e-service
> ==================================

Reviewer: ```suggestion
Progettazione dei servizi digitali e degli e-service
=====================================
```

> ==================================

Reviewer: ```suggestion
Progettazione dei servizi digitali e degli e-service
=======================================
```

Progettazione dei servizi digitali e degli e-service

> =====================================

Reviewer: ```suggestion
==========================================================
```

Un servizio consiste in un’attività o in una serie di attività svolte 
in uno scambio tra un fornitore e un cliente.

Il CAD all'art.64-bis comma 1-bis prevede che le PA, i fornitori di identità digitali e i prestatori dei servizi fiduciari qualificati, "in sede di evoluzione, progettano e sviluppano i propri sistemi e servizi in modo da garantire l'integrazione e l'interoperabilità tra i diversi sistemi e servizi (..) espongono per ogni servizio le relative interfacce applicative" (API). Lo stesso art.64-bis al comma 1-quater prevede che le PA "rendono fruibili tutti i loro servizi anche in modalità digitale". 

Se l’efficacia di un servizio pubblico è determinata dal soddisfacimento della norma, l’efficienza 
può essere incrementata utilizzando gli strumenti propri della Information 

> and Communication Technology (ICT) sfruttando anche la modalità digitale.

Author: volevo dire che attraverso l'ICT il servizio può cos' offrire anche la modalità digitale. E' l'art.64-bis al comma 1-quater che dice "anche in modalità digitale". cmq non è molto rilevante, se sta scritto prima

> and Communication Technology (ICT) sfruttando anche la modalità digitale.

Reviewer: ```suggestion
and Communication Technology (ICT) in coerenza col principio "digital first".
```

I processi che utilizzano gli strumenti ICT sono qui detti processi 
digitali.

Un e-service è un servizio erogato via Internet o attraverso una rete 
privata tramite un processo digitale in cui sono coinvolti erogatori e 

> fruitori. Gli e-service sono una particolare categoria di servizi in rete basati su interfacce applicative (API).

Reviewer: ```suggestion
fruitori.
Gli e-service sono una particolare categoria di servizi in rete basati su interfacce applicative (API).
```
